.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and test
+
+on:
+  push:
+    branches: [ "develop", "develop_1.x" ]
+  pull_request:
+    branches: [ "develop", "develop_1.x" ]
```

Comment:
Can we add `main_1.x` here so we can trigger a build when we want to merge a release candidate into that branch?

Reply:
Sure, will do 👍